src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -1405,12 +1409,36 @@ void AutomationInstrumentClipView::editPadAction(bool state, uint8_t yDisplay, u
 					    getParameterKnobPos(modelStackWithParam, getPosFromSquare(leftPadSelectedX)) + kKnobPosOffset;
 					indicator_leds::setKnobIndicatorLevel(0, knobPos);
 
```

Comment:
for the right pad in a long press, the value to display is the value at the very last node, previously I was getting it at the starting node of the last pad.

leveraged the same code I use elsewhere in this view to get the knob position of the last node.